internal/controller/instaslice_controller.go

```diff
@@ -74,10 +74,8 @@ const requeueDelay = 2 * time.Second
 //+kubebuilder:rbac:groups=apps,resources=daemonsets,verbs=get;list;watch;create;update;delete
 
 func (r *InstasliceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-
-	emulatorMode := os.Getenv("EMULATOR_MODE")
-	log.FromContext(ctx).Info("EMULATOR_MODE ", "EMULATOR_MODE", emulatorMode)
-
+  log := logr.FromContext(ctx)
```

Comment:
seems like a formatting issue.

Reply:
This was taken care of by the previous pr here https://github.com/openshift/instaslice-operator/pull/149/files

Reply:
code change looks fine just not go fmt'ed... Looks like the force push cleared it up.